ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -276,9 +275,12 @@ if [[ "$ENABLE_AUDIT_LOG" == "true" ]]; then
 fi
 
 BILLING_ACCOUNT_SWITCH=""
-if [[ ! -z "$BILLING_ACCOUNT" ]]; then
+if [[ "$ENABLE_BILLING_ACCOUNT" == "yes" ]]; then
+  BILLING_ACCOUNT=$(head -n 1 ${CLUSTER_PROFILE_DIR}/aws_billing_account)
```

Comment:
Is that file existing somehow ?

Reply:
Sorry for missing both of your comments. Currently, `aws_billing_account` likes `awscred` and we use the script error message as a tip. Document it in `ref` is a good suggestion to improve the user experience of this feature. I will put this in my backlog. Thanks.